worker/scripts/install.ps1

```
@@ -0,0 +1,374 @@
+# ============================================================
+# Run Worker
+# ============================================================
+function Start-Worker {
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$BinaryPath,
+        [Parameter(Mandatory = $true)]
+        [hashtable]$Config
+    )
+
+    $args = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)